docs/downstream-integrations/api/authentication.md

```diff
@@ -24,30 +24,130 @@ The AE token must be passed in the HTTP `Authorization` (case insensitive) reque
+**account:read**
 
-You should receive a confirmation page when you sign in successfully via your authentication platform of choice. For instance, an authenticated wallet connection confirmation page would look like this:
+- Read all the wallet address(es) you connected with Collab.Land and their balances.
+- Read basic user information including username, id , pfp.
+- See communities you are in and administrating.
 
-![Sign in with Discord 2](../imgs/connected.png) -->
+**access-control:read**
+Read user roles in a certain community
+
+**asset-ownership:read**
+Check ownership for the given account for the given token assets
+
+**gm-pass:read**
+Read gmPASS address and DID info
+
+**read-vcs**
+Read all public verifiable credentials issued to you
+
+**request-vcreds**
+Issue verifiable credentials to your DID (gmPASS)
+
+**kudos:read**
+Read certain type public verifiable credentials issued to you
+
+:::success Please be aware that these scopes are currently being revised and will soon be updated to ensure accurate naming conventions and consistency.
+:::
+
+## Login with Collab.Land (LWC) Flow
+
+To initiate the OAuth2 flow, direct users to the Collab.Land OAuth2 authorization endpoint:
+
+```bash
+https://api.collab.land/oauth2/authorize
+```
+
+Include the following parameters in the URL:
+
+- **`response_type`**: Set this to **`code`** for the authorization code flow or **`token`** for the implicit token flow.
+- **`client_id`**: The client ID obtained during the application registration process.
+- **`redirect_uri`**: The URI to which Collab.Land will redirect the user after authentication. Must be one of your registered redirect URIs. Register your localhost URIs for testing.
+- **`scope`**: The desired scopes requested by your application, separated by spaces.
+- **`state`**: An opaque string that the client application provides, the value will be sent back as-is for the callback through a redirect uri.
+
+For example, to initiate the authorization code flow:
+
+```bash
+https://api.collab.land/oauth2/authorize?response_type=code&client_id=YOUR_CLIENT_ID&redirect_uri=YOUR_REDIRECT_URI&scope=scope1+scope2&state=UNIQUE_STATE_STRING
+```
+
+Or, for the implicit token flow:
+
+```bash
+https://api.collab.land/oauth2/authorize?response_type=token&client_id=YOUR_CLIENT_ID&redirect_uri=YOUR_REDIRECT_URI&scope=scope1+scope2&state=UNIQUE_STATE_STRING
+```
+
+Upon calling the authorization endpoint, users will be prompted to log in with their Collab.Land account and grant the requested permissions to your application.
+
+## Obtain Access Tokens
+
+Collab.Land supports two OAuth2 flows: `Authorization code flow` and `Implicit token flow`. The flow you choose depends on your application's requirements. For example, if your application has a backend that can securely store the client secret, you should use the authorization code flow. If your application is a client-side application, you can use the implicit token flow.
+
```

Comment:
Maybe we should link to the oAuth2 spec for the two flows (we actually support more flows, including client-credentials)

Reply:
What's the link to the oauth2 spec?